song_story/prompt-generation/utils.py

```diff
@@ -0,0 +1,7 @@
+def encode(batch, tokenizer, max_len):
+    """
+    This function takes in a batch and a tokenizer and outputs encoded batch
+    """
```

Comment:
why to use a new file when you have no imports and a function

Reply:
This function is being imported into two different files for training and inference, hence created a separate file for it